crates/bevy_audio/src/audio_source.rs

```diff
@@ -8,7 +8,7 @@ use std::{io::Cursor, sync::Arc};
 #[derive(Debug, Clone, TypeUuid)]
 #[uuid = "7a14806a-672b-443b-8d16-4f18afefa463"]
 pub struct AudioSource {
-    pub bytes: Arc<[u8]>,
+    bytes: Arc<[u8]>,
```

Comment:
why removing the `pub`? doesn't it allow someone to generate an audio source instead of loading it from a file?

Reply:
Good point. I made it public again.